.travis.yml

```diff
@@ -4,4 +4,4 @@ rvm:
   - "2.4.1"
 sudo: false
 cache: bundler
-before_install: gem install bundler -v 1.10.3
+before_install: gem install bundler -v 2.0.1
```

Comment:
is this necessary anymore?  Does it work without it?  I think we only had it because we didn't want a new bundler to be installed... but we are specifying a new bundler so maybe it's no longer needed?

Reply:
Oh, you mean just remove the line completely? I can try it. Wondered why it was there.

Reply:
Ok, took that out, though it required removing it as a dev dependency, which should be fine (you usually don't specify that anyway). I also bumped the rvm versions to match manageiq.

Reply:
@jrafanie ok, green again. :)